data/registry/collector-receiver-database.yml

```diff
@@ -0,0 +1,14 @@
+title: OTel based Data Collector for Databases
```

Comment:
```suggestion
# cSpell:ignore ODCD
title: OTel based Data Collector for Databases (ODCD)
```

Reply:
@svrnm - in terms of trademark issues, which we were discussing recently, can third parties name their tools "OTel something-or-other"?

Reply:
Awaiting @svrnm's feedback on the title before marking this as resolved.

Reply:
@svrnm @chalin 
Is there any update of this PR?

Reply:
As far as I remember "otel" is not the issue, as there is no trademark for it, but without being the SME on that "... based" clearly indicates that this tool is "powered by"